### PR TITLE
Update the MLaaS Documentation

### PR DESCRIPTION
**Pull request by dianajlailaty**

### CODE REVIEW

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 1]
```diff
@@ -356,85 +356,25 @@ Once a MLaaS instance is up and running, it could be used for:
 Using MLaaS, you can easily deploy and use any machine learning model as a REST Web Service on a physical or a virtual compute host on which there is an available Proactive Node. Going through the Proactive Scheduler,
 you can also trigger the deployment of a specific VM using the Resource Manager elastic policies, and eventually, deploy a Model-Service on that specific node.
 
-The following deployment steps are similar wherever you need to deploy and use your model:
+In the following subsections, we will illustrate the life cycle of MLaaS instance starting from the generic service instance,
```
Comment (mklkun) [inline src/docs/MLOS/MLOSUserGuide.adoc:359]: of MLaaS instance => of an (or a) MLaaS instance

Comment (mklkun) [reply src/docs/MLOS/MLOSUserGuide.adoc:359]: starting from the generic service instance => maybe you should put the same form as the next sentences. A suggestion:
from starting the generic service instance, deploying a specific model, pausing it, to deleting the instance. We will also talk about how these ...

**Comment by andrewssobral** [comment]
@dianajlailaty please give me some time to review this pull request before merging, I let you first fix the remarks from @mklkun and anyone else before, thank you

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 2]
```diff
@@ -463,209 +400,328 @@ Execute the Workflow by setting the different workflow's variables as described
...
+| `DOCKER_IMAGE`
+| The Docker image that contains all the dependencies and libraries needed by the model specific AI model to be deployed.
+| String (default=link:https://hub.docker.com/r/activeeon/model_as_a_service/[activeeon/model_as_a_service])
 |===
 
-=== Deploy a Specific AI Model
+==== Deploy a Specific AI Model
 
-There are three ways to deploy a model in the running model_server (i.e., where you will deploy, update and call your specific AI model) using MLOS:
+You can also deploy a specific AI model directly from the the link:https://try.activeeon.com/Studio[Studio Portal].
```
Comment (mklkun) [inline src/docs/MLOS/MLOSUserGuide.adoc:410]: the the => the

**Pull request merged**